projects/1.20/assets/goldelight/goldelight/lang/zh_cn.json

```
@@ -0,0 +1,313 @@
+  "farmersdelight.tooltip.golden_dog_food.when_feeding": "喂给驯服的狼时:",
+  "farmersdelight.tooltip.golden_horse_feed.when_feeding": "喂给驯服的坐骑时:",
+  "farmersdelight.tooltip.golden_hot_cocoa": "保护您免受冻结",
+  "farmersdelight.tooltip.golden_melon_juice": "轻微瞬间治疗",
+  "farmersdelight.tooltip.golden_milk_bottle": "清除一个效果",
+  "goldelight.advancement.avarice": "贪婪的化身",
+  "goldelight.advancement.avarice.desc": "吃下一顿黄金套餐！",
+  "goldelight.advancement.drink_golden_hot_cocoa": "无论如何，寒冷从未困扰过我",
+  "goldelight.advancement.drink_golden_hot_cocoa.desc": "热饮可以防止你冻僵。",
+  "goldelight.advancement.eat_golden_shepherds_pie": "不再需要医生",
+  "goldelight.advancement.eat_golden_shepherds_pie.desc": "膳食比药物更强",
+  "goldelight.advancement.enchanted_midas_knife": "神话归来",
+  "goldelight.advancement.enchanted_midas_knife.desc": "拔出王者的力量。那些爱黄金的人会帮助你。",
+  "goldelight.advancement.get_all_gd_crates": "很难区分",
+  "goldelight.advancement.get_all_gd_crates.desc": "获取所有箱装金作物",
+  "goldelight.advancement.get_ancient_gold_dust": "被诅咒的碎片",
+  "goldelight.advancement.get_ancient_gold_dust.desc": "从海洋废墟或埋藏的宝藏中获取远古金粉",
+  "goldelight.advancement.get_ancient_gold_upgrade_smithing_template": "史密斯的宝藏",
+  "goldelight.advancement.get_ancient_gold_upgrade_smithing_template.desc": "从金字塔获取黄金锻造模板",
+  "goldelight.advancement.get_gd_seed": "黄金庄稼",
+  "goldelight.advancement.get_gd_seed.desc": "获取一个金色种子",
+  "goldelight.advancement.get_golden_mushroom_colony": "帝王蘑菇",
+  "goldelight.advancement.get_golden_mushroom_colony.desc": "剪掉一个完全成熟的金蘑菇菌落。要种植它们，你需要一个非常非常肥沃的土壤......",
+  "goldelight.advancement.get_midas_knife": "国王之手",
+  "goldelight.advancement.get_midas_knife.desc": "它看起来是锁定的",
+  "goldelight.advancement.get_midas_potion": "金突变",
+  "goldelight.advancement.get_midas_potion.desc": "将诅咒龙息用于牛、鸡或蜜蜂",
+  "goldelight.advancement.give_golden_dog_food": "强大的朋友",
+  "goldelight.advancement.give_golden_dog_food.desc": "给驯服的狼喂食金狗粮",
+  "goldelight.advancement.give_golden_horse_feed": "比蝾螈快得多",
+  "goldelight.advancement.give_golden_horse_feed.desc": "给驯服的马喂食金马饲料",
+  "goldelight.advancement.killed_raider_with_rotten_golden_tomato": "腐朽的战争",
+  "goldelight.advancement.killed_raider_with_rotten_golden_tomato.desc": "用烂金番茄杀死袭击者",
+  "goldelight.advancement.place_golden_compost": "这是分解的吗？",
+  "goldelight.advancement.place_golden_compost.desc": "放下一些金堆肥。它不会满足于普通蘑菇。",
+  "goldelight.advancement.place_golden_cooking_pot": "以金换金",
+  "goldelight.advancement.place_golden_cooking_pot.desc": "你可以把食物的效果拉出来！",
+  "goldelight.advancement.place_golden_soil": "分解者的严肃性",
+  "goldelight.advancement.place_golden_soil.desc": "自然的力量是惊人的！",
+  "goldelight.advancement.root": "金制乐事",
+  "goldelight.advancement.root.desc": "一个黄金世界在等着你！",
+  "goldelight.container.golden_cooking_pot": "金厨锅",
+  "goldelight.container.golden_cooking_pot.heated": "加热",
+  "goldelight.container.golden_cooking_pot.not_heated": "需要来自下方的灵魂热量",
+  "goldelight.container.golden_cooking_pot.served_on": "服务于: %s",

```

```suggestion
  "goldelight.container.golden_cooking_pot.served_on": "需盛装于：%s",
```
农夫乐事用词
serve一词有“上菜”的意思